packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js

```diff
@@ -22,16 +22,19 @@ class SeleniumDockerService {
    */
   async onPrepare(config) {
     const { launcherOptions } = config;
-    const { keepAliveSeleniumDockerService } = launcherOptions || {};
+    const { disableSeleniumService, keepAliveSeleniumDockerService } = launcherOptions || {};
 
+    this.disableSeleniumService = disableSeleniumService === true;
     this.keepAliveSeleniumDockerService = keepAliveSeleniumDockerService === true;
 
-    try {
-      // Verify docker is installed.
-      await exec('docker -v');
-      await this.startSeleniumHub();
-    } catch (error) {
-      throw new SevereServiceError(error);
+    if (!this.disableSeleniumService) {
```

Comment:
This check here and in `onComplete` below is to basically mimic what [this](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/config/wdio.conf.js#L146) is doing to ignore running this service if `WDIO_DISABLE_SELENIUM_SERVICE` or `SELENIUM_GRID_URL`.

I thought about just keeping [this](https://github.com/cerner/terra-toolkit/pull/583/files#diff-f13ec26e86b6e8d765ec9020014d38600b7df8643e37a26d40000d28807e2bfbL110-L112) to continue to use ENV only for this scenario but it is inconsistent with everything else. So I went with this logic here. Any better suggestions?

Reply:
If we chose to implement the services in the terra-cli I think this issue would go away:

https://github.com/cerner/terra-toolkit/pull/583#discussion_r581276065

Reply:
Add the services to the launcher and reverted these changes since they are no longer needed: https://github.com/cerner/terra-toolkit/pull/583/commits/eb6e22e962d6bb12f197d735445237e6d2aaa5ff